cli/context/store/metadatastore.go

```diff
@@ -77,15 +80,15 @@ func (s *metadataStore) getByID(id contextdir) (Metadata, error) {
 		Endpoints: make(map[string]interface{}),
 	}
 	if err := json.Unmarshal(bytes, &untyped); err != nil {
-		return Metadata{}, err
+		return Metadata{}, fmt.Errorf("parsing %s: %v", fileName, err)
```

Comment:
This changes is masking the original error (`%v`). I don't _THINK_ we use any of these errors as sentinel errors (had a cursory glance at call-sites for this code, and I think we only use the "not found" case as special), so we _probably_ should be fine, but I'd 🤦 if I didn't comment, and regret it later.

I guess there's not much we can do in these cases, other than report "something was wrong; here's some info", so I guess we should be fine.

Only alternative would be to make this an `errdefs.InvalidParam()` (if we consider this "input is invalid", and to consistently return on of the `errdefs` types), but probably something we could still consider in a follow-up if we see a need.